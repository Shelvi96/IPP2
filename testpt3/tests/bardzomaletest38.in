NEW ID

:1:49>6:::231876
634:7>5
2>626;17998
36458>64704178
208774>9:02
1;672>07
;5:6>2;53
;72626849>6726590:9
4996778>43
334:5269;6>:
24:627:68>0871738
6;889;6::>61105;;7;9
@9:74367076332069;929:3:3792660
@4;506;06:622828955;3331::21801
@84:9;29907;23635;0468;;:91;925
@291;
@7028;127;903:
@9:498520389:;:32;;71766
@1:;511::36;3;2
@62041;082;37:15;
@32043:768::04567:30938363:
@5370;:031320;52;9;:8959:09161
@::79
@:72119819170262;870:;90
@2770652405:5
@663;58542
