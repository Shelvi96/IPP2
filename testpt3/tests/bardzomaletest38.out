20406098869825212
1057603286529326185
4809559384305303552
0
1
134225975544
14
2905
22010000100000
96145779725964540
0
3878987940
0
0
